e2e-tests/gatsby-static-image/src/pages/intrinsic.js

```diff
@@ -8,26 +8,26 @@ const FluidPage = () => (
     <div data-testid="image-fluid">
       <StaticImage
         src="../images/citrus-fruits.jpg"
-        layout="intrinsic"
```

Comment:
On the surface it's a little confusing why the file `/intrinsic.js` gets its layout changed from `intrinsic` to something else ... it seems like ... an intrinsic value to change. especially since it's kind of the opposite in the `/fluid.js` file.

Reply:
We've renamed this to constrained, the file name is legacy so I updated it. If your comment is more about the naming itself though, let me know.